README.md

```diff
@@ -48,7 +48,10 @@ No. We continue to consume Fluent Bit from its main repository. We are not forki
 
 ### Public Images
 
-Each release updates the `latest` tag and adds a tag for the version of the image.
+Each release updates the `latest` tag and adds a tag for the version of the image. A `stable` tag is also available which can be trusted that it is the latest version in which there are no high impact bugs. A release may be marked as stable if the following rules are all met:
+* It has been out for a period of time
+* The Fluent Bit version contained in the release is a patch version as the patch version indicates the release includes bug fixes
+* It has been no bugs reported which will have high impact for any of our customers
```

Comment:
Let's rephrase this a bit:

"No bugs have been reported which we expect will have high impact for AWS customers. This means bugs in the components that are most frequently used by AWS customers, such as the AWS outputs of the tail input."